From cd08272c3c97d516967446be5fd46617fee5cd58 Mon Sep 17 00:00:00 2001
From: "Carlyon W. Tung" <loongs@syuusui.com>
Date: Tue, 4 Jul 2023 00:35:05 +0800
Subject: [PATCH 18/20] tools mkimage fix build with LibreSSL

---
 tools/sunxi_toc0.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/tools/sunxi_toc0.c b/tools/sunxi_toc0.c
index bab5d17b..e9e063a4 100644
--- a/tools/sunxi_toc0.c
+++ b/tools/sunxi_toc0.c
@@ -33,6 +33,12 @@
 #define pr_err(fmt, args...)	fprintf(stderr, pr_fmt(fmt), "error", ##args)
 #define pr_warn(fmt, args...)	fprintf(stderr, pr_fmt(fmt), "warning", ##args)
 #define pr_info(fmt, args...)	fprintf(stderr, pr_fmt(fmt), "info", ##args)
+ 
+#if defined(LIBRESSL_VERSION_NUMBER)
+#define RSA_get0_n(key) (key)->n
+#define RSA_get0_e(key) (key)->e
+#define RSA_get0_d(key) (key)->d
+#endif
 
 struct __packed toc0_key_item {
 	__le32  vendor_id;
-- 
2.37.3.windows.1

