From d44c788ad0e7f0d2109aeaf2615ef8ab074d113c Mon Sep 17 00:00:00 2001
From: "Carlyon W. Tung" <loongs@syuusui.com>
Date: Mon, 3 Jul 2023 16:30:56 +0800
Subject: [PATCH] Ombres

---
 arch/arm/mach-rockchip/rk3568/Kconfig | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm/mach-rockchip/rk3568/Kconfig b/arch/arm/mach-rockchip/rk3568/Kconfig
index 4e7c02cc..e4d59d1b 100644
--- a/arch/arm/mach-rockchip/rk3568/Kconfig
+++ b/arch/arm/mach-rockchip/rk3568/Kconfig
@@ -6,6 +6,11 @@ config TARGET_EVB_RK3568
 	help
 	  RK3568 EVB is a evaluation board for Rockchp RK3568.
 
+config TARGET_MONOPI_R5S_RK3568
+	bool "MonoPi R5S Board"
+	help
+	  MonoPi R5S Ombres is a board for Rockchp RK3568.
+
 config ROCKCHIP_BOOT_MODE_REG
 	default 0xfdc20200
 
@@ -18,6 +23,7 @@ config SYS_SOC
 config SYS_MALLOC_F_LEN
 	default 0x2000
 
+source "board/ombres/monopi-r5s-rk3568/Kconfig"
 source "board/rockchip/evb_rk3568/Kconfig"
 
 endif
-- 
2.37.3.windows.1

